packages/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/types/FlutterSignUpRequest.kt

```diff
@@ -24,32 +24,20 @@ import com.amplifyframework.AmplifyException
 import com.amplifyframework.auth.AuthUserAttribute
 import com.amplifyframework.auth.AuthUserAttributeKey
 import com.amplifyframework.auth.cognito.options.AWSCognitoAuthSignUpOptions
-import java.lang.reflect.Method
 
 data class FlutterSignUpRequest(val map: HashMap<String, *>) {
```

Comment:
I wasn't sure why it was necessary to change sign up related files, I guess it made sense to do this as a small refactor based on utilities you added?

Reply:
Yeah. At the very least I wanted to pull the list of standard attributes into a single location so that we are not updating it in two places.

Also, the current approach of using reflection has a bug. There is a standard attribute called "zoneinfo" (all lowercase according to how it is defined in cognito) but the Android SDK defines the method for creating that attribute key as `AuthUserAttributeKey.zoneInfo()` (treating zone and info and two words, camel cased). That means that currently there is no way to add the "zoneinfo" standard attribute with the flutter sdk. If a customer tries to add "zoneinfo" it will throw an error because of the difference in capitalization, but any other spelling ("zone_info", "zoneInfo", etc.) would be added as custom attribute.

Reply:
This is the exception that is currently thrown when attempting to add "zoneinfo" today.

```
kotlin.TypeCastException: null cannot be cast to non-null type java.lang.reflect.Method
```

Reply:
thanks for info